### PR TITLE
Add skywalking extension provider to demo profile

### PR DESCRIPTION
**Pull request by kezhenxu94**
**Please provide a description of this PR:**

### CODE REVIEW

File: manifests/profiles/demo.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@ spec:
       envoyOtelAls:
         service: otel-collector.istio-system.svc.cluster.local
         port: 4317
+    - name: skywalking
+      skywalking:
+        service: skywalking-oap.istio-system.svc.cluster.local
```
Comment (zirain) [inline manifests/profiles/demo.yaml:13]: is this should be `tracing.istio-system.svc.cluster.local`?
https://github.com/istio/istio/blob/f98481879ae4f869478ad900af6c96db9b8593bd/samples/addons/extras/skywalking.yaml#L37

**Review by zirain — COMMENTED** [review]
lgtm, leave to ENV maintainers

**Comment by kezhenxu94** [comment]
Gentle ping~

**Comment by kezhenxu94** [comment]
🥲 it’s marked as stale. Any chance a maintainer can review?

**Comment by ericvn** [comment]
Should this require a release note?

**Comment by kezhenxu94** [comment]
@zirain can you please reopen this PR?

**Review by zirain — APPROVED** [review]

**Pull request merged**